dcspy/models.py

```diff
@@ -490,6 +511,7 @@ def from_control_key(cls, ctrl_key: ControlKeyData, rb_iface: str, custom_value:
             'rb_fixed_step_dec': f'{ctrl_key.name} DEC',
             'rb_set_state': f'{ctrl_key.name} CYCLE {ctrl_key.suggested_step} {ctrl_key.max_value}',
             'rb_custom': f'{ctrl_key.name} CUSTOM {custom_value}',
+            'rb_button': f'{ctrl_key.name} BUTTON {ctrl_key.name} 1|{ctrl_key.name} 0|',
```

Comment:
@sleighzy You can take into consideration set: `ctrl_name 1|ctrl_name 0|` here or maybe directly in `BasicAircraft._load_plane_yaml()` and then keep here `'rb_button': f'{ctrl_key.name} BUTTON'`

I need think about it....